README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
> I think we should just create a tag for the day Ecma published 262 6th edition. That's the tests for 6th edition. Anything else is the tests for future editions. Our coverage might not be very good for 6th edition but that's fine.

Although it looks simple, that would be very ineffective. We still lack the coverage for important features of ES6 today and a tag from the day it was published would not give a reasonable value to use it.

With less effort, a filter using the frontmatter tags is still more effective regarding what we still have today.

The best case scenario would be having all the tests before each final release is published. Even if it demands an extra time from feature champions to support the tests before each spec goes to Stage 4. That won't be a strategy to save the tests for ES2015, but would help to avoid the same issue when the next version is released.